### ISSUE TITLE
Semantic Frequency Prompt

### ISSUE THREAD

**Issue by SummerTide** [Gumpest/FreeKD#1]
Dear authors, thanks for your kindly sharing of code. Your work is very impressive and helpful. But when I try to understand your work, I found that I didn’t know how to get the weights of mask_modules in FreeKDLoss for my own dataset and model. Is there something else I missed? Could you share more details related with semantic frequency prompt? Looking forward your reply.

**Comment by Gumpest** [comment]
Thank you for your attention and interest in FreeKD. 😊

The `mask_modules` corresponds to the `Frequency Prompt` in the paper, and its checkpoint is here: 
https://github.com/Gumpest/FreeKD/blob/main/configs/distill/freekd/freekd_retinanet_r101-retinanet_r50_coco.py#L149.

In FreeKDLoss, we will load the checkpoint at: https://github.com/Gumpest/FreeKD/blob/main/mmrazor/models/losses/freekd.py#L126.

If you want to customize your frequency prompt, you need to insert the `MaskModule` https://github.com/Gumpest/FreeKD/blob/main/mmrazor/models/losses/freekd.py#L26
into the teacher model and fine-tune, and save the `MaskModule` checkpoint for the next distillation.

**Comment by SummerTide** [comment]
Thanks for your reply, I'll try to understand. :3

**Comment by SummerTide** [comment]
Dear author, I found that you used the Jaccard coefficient to train the mask in your paper, but I did not find it in the code. 
[https://github.com/Gumpest/FreeKD/blob/main/mmrazor/models/losses/freekd.py#L11](https://github.com/Gumpest/FreeKD/blob/main/mmrazor/models/losses/freekd.py#L11)
Do I need to implement it myself? And how to visualize the frequency domain mask well in the spatial domain?
[https://github.com/Gumpest/FreeKD/blob/main/assests/media.png?raw=true](https://github.com/Gumpest/FreeKD/blob/main/assests/media.png?raw=true)
The visualization effect of the mask obtained at different scales after upsampling is not ideal.

**Comment by SummerTide** [comment]
And about the code
[https://github.com/Gumpest/FreeKD/blob/main/mmrazor/models/losses/freekd.py#L66](https://github.com/Gumpest/FreeKD/blob/main/mmrazor/models/losses/freekd.py#L66)
Here you named the return as cA, (cH, cV, cD). Intuitively, do they correspond to approximate, horizontal, vertical, and diagonal features? 
However, from the source code, is the return result a list of features at different scales?
[https://github.com/fbcotter/pytorch_wavelets/blob/master/pytorch_wavelets/dwt/transform2d.py#L68](https://github.com/fbcotter/pytorch_wavelets/blob/master/pytorch_wavelets/dwt/transform2d.py#L68)
Looking forward to your reply :3

**Comment by RobinCym7** [comment]
May I ask how the teacher model performs the fine-tune
